File: sample/src/main/java/com/thoughtbot/expandablerecyclerview/sample/multicheck/MultiCheckArtistViewHolder.java
package com.thoughtbot.expandablerecyclerview.sample.multicheck;

import android.view.View;
import android.widget.Checkable;
import android.widget.CheckedTextView;
import com.thoughtbot.expandablecheckrecyclerview.viewholders.CheckableChildViewHolder;
import com.thoughtbot.expandablerecyclerview.sample.R;

public class MultiCheckArtistViewHolder extends CheckableChildViewHolder {

  private CheckedTextView childCheckedTextView;

  public MultiCheckArtistViewHolder(View itemView) {
    super(itemView);
    childCheckedTextView =
        (CheckedTextView) itemView.findViewById(R.id.list_item_multicheck_artist_name);
  }

  @Override
  public Checkable getCheckable() {
    return childCheckedTextView;
  }

  public void setArtistName(String artistName) {
    childCheckedTextView.setText(artistName);
  }
}

File: sample/src/main/java/com/thoughtbot/expandablerecyclerview/sample/multicheck/MultiCheckArtistViewHolder.kt
package com.thoughtbot.expandablerecyclerview.sample.multicheck

import android.view.View
import android.widget.Checkable
import android.widget.CheckedTextView
import com.thoughtbot.expandablecheckrecyclerview.viewholders.CheckableChildViewHolder
import com.thoughtbot.expandablerecyclerview.sample.R

class MultiCheckArtistViewHolder(itemView: View) : CheckableChildViewHolder(itemView) {
    private val childCheckedTextView: CheckedTextView
    override fun getCheckable(): Checkable {
        return childCheckedTextView
    }

    fun setArtistName(artistName: String?) {
        childCheckedTextView.text = artistName
    }

    init {
        childCheckedTextView = itemView.findViewById<View>(R.id.list_item_multicheck_artist_name) as CheckedTextView
    }
}